scripts/configs/gvisor.go

```diff
@@ -0,0 +1,15 @@
+package configs
+
```

Comment:
not sure about the contents of this file.... we plan to have everything that works in the `main` branch